packages/actor-extract-links-predicates/lib/ActorExtractLinksPredicates.ts

```diff
@@ -25,7 +27,10 @@ export class ActorExtractLinksPredicates extends ActorExtractLinks {
         if (!this.checkSubject || this.subjectMatches(quad.subject.value, action.url)) {
           for (const regex of this.predicates) {
             if (regex.test(quad.predicate.value)) {
-              links.push({ url: quad.object.value });
+              links.push({
+                url: quad.object.value,
+                metadata: { producedByActor: { name: this.name, predicates: this.stringPredicates }},
```

Comment:
Not sure what would be best here, but instead of listing all predicates, would it make more sense perhaps to include the matched predicate?

Reply:
My purpose was to determine the reachability criteria used for instance Ccommon, Cldp, CsolidStorage. So I would need to know every predicate declared by the actor (to handle a case where for example two actors have predicates in common). But perhaps I could include also the matching predicate.

Reply:
Ok, let's add both then :-)